pkg/usecase/sync_archives.go

```diff
@@ -46,6 +46,11 @@ func (s *SyncArchiveUseCase) SyncArchivesWithInterval(ctx context.Context) error
 		WithName("support archive sync interval handler")
 	logger.Info(fmt.Sprintf("started regularly syncing support archives with interval %s", s.syncInterval))
 
+	if s.syncInterval == 0 {
```

Comment:
Ah okay. So if the sync is disabled the gc would not delete all data from the archives.
It is better to use the delete usecase above.